### PR TITLE
WIP-Adding Contributing.md

### PR DESCRIPTION
**Pull request by saxenaaman628**

### CODE REVIEW

**Comment by whyDontI** [comment]
I think you meant `CONTRIBUTING.md` not `CONFIGURATION` 😁

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+To run the project `yarn run dev`.
+
+## Production
+
+To do a production build `yarn run start`.
+
+
+## **3. Api Contracts**
+
+Api contracts can be found [here](https://github.com/Real-Dev-Squad/website-api-contracts/tree/main/tasks).
```
Comment (swarajpure) [inline CONTRIBUTING.md:32]: NIT: `API` instead of `Api`

**Review by whyDontI — CHANGES_REQUESTED** [review]
Good work @saxenaaman628 😃 
Can you also add how we perform GitHub auth using [this](https://github.com/login/oauth/authorize?client_id=c4a84431feaf604e89d1) link?

`https://github.com/login/oauth/authorize?client_id=c4a84431feaf604e89d1`

**Comment by saxenaaman628** [comment]
But we are already making local system secure. Do we still need to add those steps.

**Pull request merged**